app/models/donation.rb

```diff
@@ -15,4 +15,12 @@ def subtotal(quantity)
   def self.active
     where(active: true)
   end
+
+  def display_active
+    if active
+      "Active"
+    else
+      "Retired"
+    end
+  end
 end
```

Comment:
@edilenedacruz 

Here's a pretty great [Ruby style guide](https://github.com/bbatsov/ruby-style-guide). This if..else statement works, but it could be refactored to a single line. Check out the style guide to see how.